app/uk/gov/hmrc/helptosavefrontend/views/email/select_email.scala.html

```diff
@@ -59,41 +56,38 @@ <h2 class="heading-medium error-summary-heading" id="error-summary-invalid-email
-          <div class="@{"form-group" + {if(hasInvalidEmail) " form-group-error" else ""}}">
-            <label class="form-label-bold" for="new-email">@messages("hts.select-email.enter-new-address-label")</label>
-
-            <span class="form-hint" id="new-email-hint">@Html(Messages("hts.select-email.enter-new-address-hint"))</span>
-            @if(hasInvalidEmail) { <span class="error-message" id="new-email-error">@email_form_validation_error_message(form, "new-email")</span> }
-
-            <input class="form-control @if(hasInvalidEmail){form-control-error}" name="new-email" type="email" id="new-email" aria-describedby="new-email-hint @if(hasInvalidEmail || hasNotSelected){ new-email-error}" @if(form.data.get("new-email").nonEmpty){ value="@form.data.get("new-email")" }>
+          <div class="govuk-radios" data-module="govuk-radios">
+            <div class="govuk-radios__item">
+            <input class="govuk-radios__input" id="registered-email" name="email" type="radio" value="Yes" @if(!hasNotSelected && newEmail.isDefined && newEmail.exists(_.isEmpty)){ checked }>
+            <label class="govuk-label govuk-radios__label" for="registered-email">@email</label>
           </div>
-          <p>@messages("hts.select-email.enter-new-address-body-text")</p>
-        </div>
-
+          <div class="govuk-radios__item">
+              <input class="govuk-radios__input" id="add-new-email" name="email" type="radio" value="No" data-aria-controls="collect-new-email" @if(hasInvalidEmail || (newEmail.isDefined && newEmail.exists(_.nonEmpty))){ checked }>
+              <label class="govuk-label govuk-radios__label" for="add-new-email">@messages("hts.select-email.select-new-address")</label>
+          </div>
+          <div class="@{"govuk-radios__conditional govuk-radios__conditional--hidden" + {if(hasInvalidEmail) " govuk-panel-with-error" else ""}}" id="collect-new-email">
+              <div class="@{"govuk-form-group" + {if(hasInvalidEmail) " govuk-form-group-error" else ""}}">
+                <label class="govuk-label" for="new-email"><strong>@messages("hts.select-email.enter-new-address-label")</strong></label>
+                <span class="govuk-hint" id="new-email-hint">@Html(Messages("hts.select-email.enter-new-address-hint"))</span>
+                @if(hasInvalidEmail) { <span class="govuk-error-message" id="new-email-error">@email_form_validation_error_message(form, "new-email")</span> }
+                <input class="govuk-input govuk-!-width-one-half @if(hasInvalidEmail){form-control-error}" name="new-email" type="email" id="new-email" aria-describedby="new-email-hint @if(hasInvalidEmail || hasNotSelected){ new-email-error}" @if(form.data.get("new-email").nonEmpty){ value="@form.data.get("new-email")" }>
+              </div>
+            <p class="govuk-label">@messages("hts.select-email.enter-new-address-body-text")</p>
+         </div>
+      </div>
```

Comment:
⛏️ I wish more substantial changes like this were separated into its own commit.

Reply:
So do you want me to revert these changes and do a new pr for the same.

Reply:
No need to create a separate PR, just _commit_ (you could do an interactive rebase + edit/partial staging and split this huge commit into two).

That's just a wish though, I've looked at it, and it seems OK. It'd just be easier to review (commit title is a bit misleading). Keep it in mind next time.